### ISSUE TITLE
Duplicate function name, for init function

### ISSUE THREAD

**Issue by fzerorubigd** [go-lang-plugin-org/go-lang-idea-plugin#1303]
I get error for init function if I have more than one init in a package, but base on godoc : 
https://golang.org/doc/effective_go.html#init

> Finally, each source file can define its own niladic init function to set up whatever state is required. (Actually each file can have multiple init functions.) And finally means finally: init is called after all the variable declarations in the package have evaluated their initializers, and those are evaluated only after all the imported packages have been initialized.

**Comment by dlsniper** [comment]
Duplicate of #1213. Fixed in branch and will be visible in next release. Thanks
